src/train.py

```diff
@@ -40,7 +58,13 @@ def train(args):
 
             character, mel, mel_input, pos_text, pos_mel, _ = data
 
-    raise Exception("TODO: Implement")
+        for batch in dataloader:
+            # choose loss function here!
+            model.decode(model.encode(batch))
```

Comment:
This can be inside the for loop above ^, where the dataloader is wrapped in tqdm.

Reply:
Yeah this is just kinda boilerplate rn, I imagine it will change a lot once we get the data/batches setup for the task

Reply:
The main focus of the PR isn't the training code, because I don't really have that ready to go.  It is more of the models changes / the RNN encoder decoder

Reply:
That's true. I think we should try to not clutter code if it's not necessary.